ui/main/src/app/modules/share/light-card/light-card.component.ts

```diff
@@ -113,11 +122,25 @@ export class LightCardComponent implements OnInit, OnDestroy {
         }
     }
 
+    private computeGroupedCardsIcon() {
+        this.showGroupedCardsIcon = this.groupedCardsService.isParentGroupCard(this.lightCard);
+    }
+
+    getGroupedChildCards() {
+        return this.groupedCardsService.getChildCardsByTags(this.lightCard.tags);
```

Comment:
to be call only if group option is activated to avoid unnecessary processing

Reply:
(Almost) Same as above, although an unnecessary sort() is called on the card tags for one card.

Again: if you feel it improves readability I can add the check.

Reply:
ok , you are right